src/Bridge/Doctrine/Common/Filter/OrderFilterTrait.php

```diff
@@ -52,6 +52,10 @@ public function getDescription(string $resourceClass): array
                 'property' => $propertyName,
                 'type' => 'string',
                 'required' => false,
+                'enum' => [
+                    strtolower(OrderFilterInterface::DIRECTION_ASC),
```

Comment:
Actually the order filter parameter could also be used without a value, e.g. `GET /books?order[datePublished]`

It'd use the default order configured for that property. (I don't like it, but it's something that's supported.)

See Behat test scenario here: https://github.com/api-platform/core/blob/cfa8f34d0d5779e0d7e4805ae0d1062afd699aab/features/doctrine/order_filter.feature#L244

Reply:
It can be documented like this but could be used without value too, doesn't it?

Reply:
Enum usually means you must use one of the specified values in the list. Haha...

Reply:
So, there is `allowEmptyValue` in OpenAPI v3. We could probably add that for other filters too, where applicable. But perhaps in another PR.

Reply:
So this PR could be merged as is.

Reply:
@alanpoulain See the other comment.